index.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+  <link rel="stylesheet" href="mobile.css">
+</head>
+<body>
```

Comment:
- [ ] OPTIONAL
Some contents under various sections are **not evenly centered**. Kindly ensure that the **spacing** (padding/ margin) on both the extreme left and right sides should be **equal**, this should be **consistent** with all the sections

**Your page layout:**
![image](https://user-images.githubusercontent.com/12745474/137305603-0c1c466b-02b9-4e31-9d62-1f8cafd71266.png)

**The original page layout:**
![image](https://user-images.githubusercontent.com/12745474/137305881-3addc139-f9ae-4d93-838f-0695d8a9c26f.png)